### PR TITLE
feat: 🎸 Added onChange handler to multiselect component SSC-107

### PR DESCRIPTION
**Pull request by Chris-Boyle**
✅ Closes: #53

Added ability for consumer to pass onChange method.

Loom: https://www.loom.com/share/60d6eebb6f8a485395a9d0f66d858ac0

### CODE REVIEW

File: src/components/SQForm/SQFormMultiSelect.js [hunk 1]
```diff
@@ -137,7 +139,7 @@ function SQFormMultiSelect({
           input={<Input disabled={isDisabled} name={name} />}
           value={field.value}
           onBlur={handleBlur}
-          onChange={handleMultiSelectChange}
+          onChange={handleMultiSelectChange && handleChange}
```
Comment (20BBrown14) [inline src/components/SQForm/SQFormMultiSelect.js:142]: With this change `handleMultiSelectChange` will never get called, right? Maybe I'm missing something.

Comment (SeanGroff) [reply src/components/SQForm/SQFormMultiSelect.js:142]: If we add a custom `onChange` prop to this component we want the original `handleMultiSelectChange` method to be called. Then if an `onChange` method is passed to the component, call it after the `handleMultiSelectChange` method has executed.

This component is imperatively calling setFieldValue already so I believe we wouldn't use the `handleChange` help method like we would in fields that are provided a single value.

I think what we need to do here is after `setFieldValue(name, values);` is called, add a new line with `onChange && onChange(value);`

Comment (Chris-Boyle) [reply src/components/SQForm/SQFormMultiSelect.js:142]: Good call.  Overlooked that by trying to use the useForm hook.  @SeanGroff and @20BBrown14 I have updated the code.
image

**Review by 20BBrown14 — APPROVED** [review]

**Review by SeanGroff — APPROVED** [review]
LGTM nice work @Chris-Boyle 🤘

**Pull request merged**